test/korma/test/core.clj

```diff
@@ -730,6 +750,14 @@
               "dry run :: SELECT \"state2\".* FROM \"state2\" WHERE (\"state2\".\"id\" = ?) :: [1]\n")
          (with-out-str (dry-run (select address2 (with state2)))))))
 
+(defentity state2-ck (transform identity) (pk :id :id2))
+(defentity address2-ck (belongs-to state2-ck (fk :sid :sid2)))
+
+(deftest test-belongs-to-with-transform-fn-for-subentity-composite-key
```

Comment:
test composite-key handling with belongs-to relationship and transform fn for subentity